crates/holochain/src/conductor/p2p_store.rs

```diff
@@ -306,6 +310,40 @@ pub fn query_agent_info_signed(
     Ok(out)
 }
 
+/// Get the peer density an agent is currently seeing within
+/// a given [`DhtArc`A.]
+pub fn query_peer_density(
```

Comment:
it looks like this isn't used anywhere yet?

Right now looks like this could count expired entries as well, should we purge the db before counting?

Reply:
Yeh I'm not actually sure where to use it yet :)
I could filter the expired but I'm hesitant to open a write for this call?

Reply:
that makes sense, yeah we can filter it, and periodically have the actual purge call separately.